db/migrate/20190502020249_add_lock_author_id_to_commits.rb

```diff
@@ -0,0 +1,5 @@
+class AddLockAuthorIdToCommits < ActiveRecord::Migration[5.2]
+  def change
+    add_column(:commits, :lock_author_id, :integer, limit: 4)
```

Comment:
`:integer`? 

Is `users.id` integer as well? If so we should prbably migrate to bigint someday, now Rails default.

Reply:
I matched `lock_author_id` on `stacks`. Was also surprised.

Reply:
Ok, disregard then.